library/Fission/User/Query.hs

```diff
@@ -9,5 +9,5 @@ import Fission.User.Types
 
 -- | Find a user by their account secret
 bySecret :: Text -> Row s User -> Col s Bool
-bySecret secret user = user `is'` #_active
-                   .&& user ! #_secretDigest .== text secret
+bySecret secret user = (user `is'` #_active)
+                   .&& (user ! #_secretDigest .== text secret)
```

Comment:
If we're going for parsing clarity, we can also do this:

```haskell
bySecret :: Text -> Row s User -> Col s Bool
bySecret secret user = isActive .&& matchesSecret
  where
    isActive      = user `is'` #_active
    matchesSecret = user ! #_secretDigest .== text secret
```

I don't mind either way; just outlining your options 😁  They're both totally readable.